index.html

```diff
@@ -139,13 +130,18 @@ <h1 id="history_header">Last Records:</h1>
             <button type="button" class="btn new-user">Select new user</button>
             <button type="button" class="btn view-setup">Setup game</button>
         </div>
+
+        <div id="game-over" class="hide"></div>
```

Comment:
These should not be inside the "history" div. They should probably be inside the "quiz" - coordinate with Ishi

Reply:
ah ok, will do

Reply:
I'm going to close this pull request until the quiz section of the html is up.